src/interfaces/AuthProvider.js

```diff
@@ -10,28 +11,30 @@
 
 /**
  * Interface declaration for Authentication Providers
+ *
+ * @interface AuthProvider
  */
-export interface AuthProvider {
+export class AuthProvider {
```

Comment:
why the change from an interface to class here, just curious.

Reply:
JSDocs didn't like it